src/java.desktop/share/classes/javax/swing/JTable.java

```diff
@@ -1164,7 +1164,7 @@ public Color getGridColor() {
      * @see     #setShowHorizontalLines
      */
     @BeanProperty(description
-            = "The color used to draw the grid lines.")
+            = "Whether grid lines should be drawn in between the cells.")
```

Comment:
It probably need a CSR update as this line is part of spec now

Reply:
I think we can skip a CSR for this. It is an obvious copy/paste typo and it is a bean property description, nit the javadoc spec which is correct.

Reply:
Thank you @prrace, @prsadhuk for your time and review.

@mrserb  thank you for bringing this up, I have updated the description made it inline with spec.